### ISSUE TITLE
Remove legacy docs on controllers

### ISSUE THREAD

**Issue by TarikGul** [paritytech/substrate-api-sidecar#1150]
Before we had the OpenAPI specs we would write the controller docs above the actual controller. Since we have more organized documentation within SwaggerAPI we no longer need them. This is also important to remove since it can be confusing for contributors or maintainers when adding features, or new endpoints. 

@Imod7 Thoughts?

**Comment by Imod7** [comment]
Yes, I agree! I will open a PR (and link this issue) to remove the comments from all the controller files.

**Comment by Imod7** [comment]
@TarikGul I was looking at some of the controller files in order to remove all comments and I am seeing some parts that are not in the [OpenAPI docs](https://paritytech.github.io/substrate-api-sidecar/dist/). 

- For example, in the [AccountsBalanceInfoController](https://github.com/paritytech/substrate-api-sidecar/blob/master/src/controllers/accounts/AccountsBalanceInfoController.ts#L35L56) the docs related to : 
  - The returned values and
  - The Substrate References

  are not present in the OpenAPI.

- Also, in other files there is other useful information that is also not present in OpenAPI. An example is in [Staking Payouts](https://github.com/paritytech/substrate-api-sidecar/blob/master/src/controllers/accounts/AccountsStakingPayoutsController.ts#L73L76).

Based on the above, in the comments of every controller I would opt for : 
- Adding a reference to the OpenAPI specs for the documentation of the current controller
- Removing only the duplicate information compared to OpenAPI
- Add any additional information that is not included in the OpenAPI specs
